pgrx-macros/src/rewriter.rs

```diff
@@ -35,8 +36,7 @@ pub fn item_fn_without_rewrite(mut func: ItemFn) -> syn::Result<proc_macro2::Tok
     let input_func_name = func.sig.ident.to_string();
     let sig = func.sig.clone();
     let vis = func.vis.clone();
-    let attrs = func.attrs.clone();
-
+    let mut attrs = take(&mut func.attrs);
```

Comment:
I don't know why this is passing tests, seeing as we use the entire func, including its attrs, down here: https://github.com/pgcentralfoundation/pgrx/blob/7ea24807bd5382f74cf5b4ea547452bca37203ee/pgrx-macros/src/rewriter.rs#L108

Reply:
I suppose because we apply attrs to the outer function as well, but I don't feel confident asserting that it's fine to simply lift the attrs up in all cases.

Reply:
**oh**, we were already clearing the attrs, I see.